base/reflection.jl

```diff
@@ -450,6 +457,19 @@ function functionloc(f)
     functionloc(first(mt))
 end
 
+"""
+    function_module(f) -> Module
```

Comment:
You mean it should be in  https://github.com/JuliaLang/julia/blob/master/doc/stdlib/base.rst, right?  That file is auto-generated, but running `make julia-genstdlib` doesn't do anything. What do I need to do?